cmd/vgmanager/main.go

```diff
@@ -75,8 +75,9 @@ func main() {
 	}
 
 	if err = (&vgmanager.VGReconciler{
-		Client: mgr.GetClient(),
-		Scheme: mgr.GetScheme(),
+		Client:   mgr.GetClient(),
+		Scheme:   mgr.GetScheme(),
+		Nodename: os.Getenv("NODE_NAME"),
```

Comment:
do we need `Nodename` in the `VGReconciler` object ? It s used only when we update the `status`. So we can directly call `os.Getenv("NODE_NAME")` over there.

Reply:
I don't see a problem with saving it. Similar to how we save the namespace elsewhere.

Reply:
ok.  name should be `NodeName`.

Reply:
Done